packages/app-builder-lib/src/winPackager.ts

```diff
@@ -355,7 +355,8 @@ export class WinPackager extends PlatformPackager<WindowsConfiguration> {
 
   private shouldSignFile(file: string): boolean {
     const shouldSignDll = this.platformSpecificBuildOptions.signDlls === true && file.endsWith(".dll")
-    return shouldSignDll || file.endsWith(".exe") || file.endsWith(".node")
+    const shouldSignExplicit = !!this.platformSpecificBuildOptions.signExts?.some(ext => file.endsWith(ext))
+    return shouldSignDll || shouldSignExplicit || file.endsWith(".exe")
```

Comment:
Why not make `.exe` part of the default `signExts` array?

Reply:
I was thinking of that, but since it has been long-established functionality, I didn't want to change it.

`.exe` is required for any signing to work AFAIK, so having it as default would technically work **until** someone explicitly sets an additional `signExts` value, in which case they'd need to also know that they'd still need to include `.exe`. This would require deeper knowledge of electron-builder and need to be explicitly called out in the docs, which can easily be overlooked.

Reply:
Makes sense